static/app/views/explore/components/schemaHintsList.tsx

```diff
@@ -299,3 +316,24 @@ const SchemaHintOption = styled(Button)`
     background-color: ${p => p.theme.gray100};
   }
 `;
+
+const HintTextContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  gap: ${space(0.5)};
+`;
+
+const HintName = styled('span')`
+  font-weight: ${p => p.theme.fontWeightNormal};
+  color: ${p => p.theme.textColor};
+`;
+
+const HintOperator = styled('span')`
+  font-weight: ${p => p.theme.fontWeightNormal};
+  color: ${p => p.theme.gray300};
```

Comment:
```suggestion
  color: ${p => p.theme.subText};
```
nit: just what search bar uses